modules/ROOT/pages/manage-audit-logs.adoc

```diff
@@ -0,0 +1,292 @@
+
+[source, json]
+----
+"logging": {
+             "audit": {
+                        “audit_log_file_path”: “/path/to/file.log”,
+                        “enabled”: true,
+                        “enabled_events”: [53270, 53280]
+                      }
+            }
+----
+
+Enabling the events at runtime without modifying the entire database
+
+You can enable events at runtime without modifying the entire database using the following HTTP request methods:
+
+* `GET /{db}/_config/audit`
+* `PUT /{db}/_config/audit (replace)`
+* `POST /{db}/_config/audit (upsert)`
+
+These request methods apply changes at the database level, a possible use case is that you can modify the `enabled` parameter to disable all events for the database.
+
+The examples below show two ways to specify audit events.
+
+
+The first example enables the `Public HTTP API request` and `Public API user authenticated` events.
+
+[source, json]
+----
+{
+	“enabled”: true,
+  “enabled_events”: [{“53270”: true, “53280”: true]
+}
+----
+
+If you wanted to enable one event and disable another:
+
+[source, json]
+----
+{
+	“enabled”: true,
+  “enabled_events”: [{“53270”: true, “53280”: false]
+}
+----
+
+=== Disabled Users and Roles
+
+You can filter audit events by specifying roles or users to be disabled. 
+The example configuration below shows disabling audit events for `user1` and `role1`. 
+
+[source, json]
+----
+{
+  “logging”: {
+               “audit”: {
+                          "enabled”: true,
+                          “enabled_events”: [],
+                          “disabled_users”: [
+                                              {
+                                                “name”: “user1”,
+                                                “domain”: “sgw”
+                                              }
+                                            ],
+                        },
+  “disabled_roles”: [
+                      {
+                        “name”: “role1”,
+                        “domain”: “sgw”
+                      }
+                    ],
+               },
+}  
+----
+
+The `disabled_users` field will prevent all audit events generated by the specified users from being logged.
+The `disabled_roles` field will prevent all audit events generated by the specified roles from logged.
+A use case for these fields would be to exclude certain administrative users or roles that perform a large volume of automated processes to prevent bloat of trivial events causing early rotation of the log file.
+
+Users and roles are organised into the following domains:
+
+* `sgw` - Users and Roles that are created by and operate solely within Sync Gateway.
+For more information, see xref:access-control-concepts.adoc#lbl-sgw-users[Sync Gateway defined Users and Roles]. 
+* `cbs` - Users that are are RBAC controlled.
+These are created on Couchbase Server.
+For more information, see xref:access-control-concepts.adoc#lbl-rbac-users[RBAC Users]. 
```

Comment:
It's probably worth explaining how the Couchbase Server users interact with Sync Gateway.  Sync Gateway's admin and metrics REST API are authenticated using Couchbase Server users.